x-pack/legacy/plugins/snapshot_restore/public/app/lib/authorization/components/not_authorized_section.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+interface Props {
+  sectionName: string;
+  privilegeType: string;
+  missingPrivileges: string[];
+  FormattedMessage: typeof ReactIntl.FormattedMessage;
+}
+
+export const NotAuthorizedSection = ({
+  sectionName,
+  privilegeType,
+  missingPrivileges,
+  FormattedMessage,
+}: Props) => (
+  <EuiEmptyPrompt
+    iconType="securityApp"
+    title={
+      <h2>
+        <FormattedMessage
+          id="xpack.snapshotRestore.restoreList.deniedPermissionTitle"
+          defaultMessage="You're missing {privilegeType} privileges"
+          values={{
+            privilegeType,
+          }}
+        />
+      </h2>
+    }
+    body={
+      <p>
+        <FormattedMessage
+          id="xpack.snapshotRestore.restoreList.deniedPermissionDescription"
+          defaultMessage="To view {sectionName}, you must have {privilegesCount,
```

Comment:
the messaging for missing index-level privileges is slightly different: [relevant LOC](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/restore_list/restore_list.tsx#L69)

how about changing this component so that the `title` and `body` are part of `Props`? this would make it similar to our `SectionError` and `SectionLoading` components

Reply:
Good point, I will update the component